<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QMainWindow" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>瑞士军刀</string>
  </property>
  <widget class="QWidget" name="Home">
   <widget class="QPushButton" name="hidemode">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>私隐模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>设定</string>
    </property>
   </widget>
   <widget class="QLabel" name="hostname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Loading……</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>351</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="v4">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>公网IPv4地址</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="v4add">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Loading......</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>351</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="v6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>公网IPv6地址</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="v6add">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Loading......</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="isp">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ISP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ispinfo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Loading......</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>351</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="localv4">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>本地IPv4地址</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="localv4add">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Loading......</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="localv6">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>本地IPv6地址</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="localv6add">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Loading......</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="currentMac">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>当前网卡MAC地址</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Mac">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Loading......</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="refresh">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QLabel" name="priority">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Loading……</string>
    </property>
   </widget>
   <widget class="QLabel" name="test_waring">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Loading……</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Version">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>410</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Loading……</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="tools">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>工具</string>
    </property>
    <widget class="QMenu" name="Ping">
     <property name="title">
      <string>NowPing 延迟检测</string>
     </property>
     <addaction name="PingCS"/>
     <addaction name="PingVAL"/>
     <addaction name="PINGGNT"/>
    </widget>
    <addaction name="WOL"/>
    <addaction name="track"/>
    <addaction name="NAT"/>
    <addaction name="Ping"/>
   </widget>
   <widget class="QMenu" name="helpcenter">
    <property name="title">
     <string>帮助中心</string>
    </property>
    <widget class="QMenu" name="git">
     <property name="title">
      <string>查看源码</string>
     </property>
     <addaction name="CNB"/>
     <addaction name="Github"/>
    </widget>
    <widget class="QMenu" name="issue">
     <property name="title">
      <string>问题反馈</string>
     </property>
     <addaction name="issuecnb"/>
     <addaction name="issuegithub"/>
    </widget>
    <addaction name="action"/>
    <addaction name="about"/>
    <addaction name="wiki"/>
    <addaction name="separator"/>
    <addaction name="blog"/>
    <addaction name="updatelog"/>
    <addaction name="git"/>
    <addaction name="issue"/>
    <addaction name="aboutqt"/>
   </widget>
   <addaction name="tools"/>
   <addaction name="helpcenter"/>
  </widget>
  <action name="WOL">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>WakeNow 局域网唤醒</string>
   </property>
  </action>
  <action name="track">
   <property name="text">
    <string>TrackNow 路由检测</string>
   </property>
  </action>
  <action name="NAT">
   <property name="text">
    <string>WhatsNAT 网络转换层级检测</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="updatelog">
   <property name="text">
    <string>更新日志</string>
   </property>
  </action>
  <action name="wiki">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wiki</string>
   </property>
  </action>
  <action name="PingCS">
   <property name="text">
    <string>PingCS 《反恐精英》延迟检测</string>
   </property>
  </action>
  <action name="PINGGNT">
   <property name="text">
    <string>PingGlobalNT 全球网测延迟检测</string>
   </property>
  </action>
  <action name="PingVAL">
   <property name="text">
    <string>PingVAL《无畏契约》国服延迟检测</string>
   </property>
  </action>
  <action name="CNB">
   <property name="text">
    <string>CNB</string>
   </property>
  </action>
  <action name="Github">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="issuecnb">
   <property name="text">
    <string>CNB</string>
   </property>
  </action>
  <action name="issuegithub">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="blog">
   <property name="text">
    <string>博客</string>
   </property>
  </action>
  <action name="aboutqt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>设备信息</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
